cmd/serverless-init/initcontainer/initcontainer.go

```diff
@@ -0,0 +1,116 @@
+package initcontainer
```

Comment:
We might want to exclude these files from the windows compile, since they rely on unix process group ids.